src/main/java/org/broadinstitute/hellbender/engine/filters/AmbiguousBaseReadFilter.java

```diff
@@ -15,30 +15,29 @@
     private static final long serialVersionUID = 1L;
 
     @Argument(doc = "Threshold fraction of ambiguous bases",
-            fullName = "ambigFilterFrac",
+            fullName = "maxAmbiguousBases",
             optional = true)
-    public double nFrac = 0.05;
+    public int maxAmbiguousBases = 0;
 
     public AmbiguousBaseReadFilter() {
     }
 
-    public AmbiguousBaseReadFilter(final double n_frac) {
-        this.nFrac = n_frac;
+    public AmbiguousBaseReadFilter(final int maxAmbiguousBases) {
+        this.maxAmbiguousBases = maxAmbiguousBases;
     }
 
     //Filters out reads with more than a threshold number of N's
     @Override
     public boolean test(final GATKRead read) {
-        final int N_max = (int) (read.getLength() * nFrac);
```

Comment:
I have some code dependent on this filter using the percentage of Ns. If this is strictly necessary, I suggest to add `maxAmbiguousBases` as a nullable `Integer` argument that is mutually exclusive with the fraction, that shouldn't be removed. In that case, this line will be the following and the rest of the method unchanged:

```java
final int N_max = (maxAmiguousBases  != null) ? maxAmiguousBases : (int) (read.getLength() * nFrac);
```

Does this makes sense? Another option is to allow both parameter and use `Math.min` to always filter based on the minimum. Actually this could be more desirable to being able to stablish a hard bound and once it passes use the fraction provided...